[2025-02-23T20:41:24.483+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-23T20:41:17.849041+00:00 [queued]>
[2025-02-23T20:41:24.508+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-23T20:41:17.849041+00:00 [queued]>
[2025-02-23T20:41:24.509+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-23T20:41:24.545+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2025-02-23 20:41:17.849041+00:00
[2025-02-23T20:41:24.568+0000] {standard_task_runner.py:57} INFO - Started process 2484 to run task
[2025-02-23T20:41:24.586+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'manual__2025-02-23T20:41:17.849041+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmp4k1wvvj6']
[2025-02-23T20:41:24.588+0000] {standard_task_runner.py:85} INFO - Job 90: Subtask kafka_data_stream
[2025-02-23T20:41:24.758+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-23T20:41:17.849041+00:00 [running]> on host ee299acebfe2
[2025-02-23T20:41:25.089+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_spark_dag' AIRFLOW_CTX_TASK_ID='kafka_data_stream' AIRFLOW_CTX_EXECUTION_DATE='2025-02-23T20:41:17.849041+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-23T20:41:17.849041+00:00'
[2025-02-23T20:41:40.369+0000] {logging_mixin.py:154} INFO - Dataset downloaded successfully.
[2025-02-23T20:41:40.763+0000] {logging_mixin.py:154} INFO - Dataset extracted successfully.
[2025-02-23T20:41:40.771+0000] {logging_mixin.py:154} INFO - No CSV file found in extracted data.
[2025-02-23T20:41:40.775+0000] {logging_mixin.py:154} INFO - Dataset preparation failed.
[2025-02-23T20:41:40.778+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-23T20:41:40.836+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_spark_dag, task_id=kafka_data_stream, execution_date=20250223T204117, start_date=20250223T204124, end_date=20250223T204140
[2025-02-23T20:41:40.924+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-23T20:41:41.012+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
